src/components/announcementsList/index.tsx

```diff
@@ -1,65 +1,155 @@
-import { Row } from 'antd';
-import React, { useEffect, useState } from 'react';
+import { range } from 'lodash';
+import { default as React, useState } from 'react';
 import styled from 'styled-components';
-import { getAnnouncementRows } from '../../containers/announcements/ducks/selectors';
 import { Announcement } from '../../containers/announcements/ducks/types';
-import { ORANGE } from '../../utils/colors';
+import { LINK, ORANGE } from '../../utils/colors';
+import {
+  NO_ANNOUNCEMENTS_HEADER,
+  NO_ANNOUNCEMENTS_SUBHEADER,
+} from '../../utils/copy';
 import { AnnouncementCard } from '../AnnouncementCard';
 
-const COLUMNS_PER_ROW = 3;
-const NO_ANNOUNCEMENTS_HEADER = 'There are currently no announcements!';
-const NO_ANNOUNCEMENTS_SUBHEADER = 'Come back later for future updates!';
-const NO_ANNOUNCEMENTS_HEADER_COLOR = ORANGE;
-
-const AnnouncementRow = styled(Row)`
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  margin-bottom: 20px;
-`;
 const NoAnnouncementsSubText = styled.span`
   display: block;
   text-align: center;
 `;
 
 const NoAnnouncementsText = styled(NoAnnouncementsSubText)`
-  color: ${NO_ANNOUNCEMENTS_HEADER_COLOR};
+  color: ${ORANGE};
   font-size: 36px;
   font-weight: 800;
 `;
 
+const AnnouncementsListWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-bottom: 40px;
+`;
+
+const PageNumbersWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+const PageNumber = styled.li`
+  font-size: 30px;
+  list-style-type: none;
+  cursor: pointer;
+  padding: 0px 16px;
+  border: 1px solid #d9d9d9;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin: 0px 8px;
+`;
+
+const SelectedPageNumber = styled(PageNumber)`
+  color: ${LINK};
+  border: 1px solid ${LINK};
+`;
+
+const ArrowButton = styled(PageNumber)`
+  color: #bfbfbf;
```

Comment:
Missed this before, but color constant bad

Reply:
f